app/controllers/idv/personal_key_controller.rb

```diff
@@ -88,11 +88,8 @@ def pending_profile?
     end
 
     def blocked_by_device_profiling?
-      return false unless IdentityConfig.store.lexisnexis_threatmetrix_required_to_verify
-      proofing_component = ProofingComponent.find_by(user: current_user)
-      # pass users who are inbetween feature flag being enabled and have not had a check run.
-      return false if proofing_component.threatmetrix_review_status.nil?
-      proofing_component.threatmetrix_review_status != 'pass'
+      !idv_session.profile.active &&
```

Comment:
Minor update here: Check that the profile is active rather than re-evaluating the `threatmetrix_review_status`